chore: auto publish alpha versions (#1414)
PostHog/posthog-js: README.md

    ```

5. Enjoy the new prerelease version. You can now use it locally, in a dummy app, or in the [main repo](https://github.com/posthog/PostHog).

> #### Automagically

Author: ```suggestion
#### Automagically (in progress - don't use yet!)
```

> #### Automagically (in progress - don't use yet!)

Author: ```suggestion
#### Automagically
```

> Use the release-alpha label on your PR to have an alpha version bumped and published automatically.
> Remember that these are public and folk might use them, so make sure they're not _too_ alpha 🙈

Author: ```suggestion
Use the "release alpha" label on your PR to have an alpha version published automatically. This automation currently doesn't check whether an alpha exists for the version it will try to publish. If you need to publish two alphas from one PR you'll need to fix that

Remember that these versions are public and folk might use them, so make sure they're not _too_ alpha 🙈
```
